concierge_cli/manager.py

```diff
@@ -0,0 +1,108 @@
+"""
+Concierge repository projects management CLI.
+"""
+from gitlab import Gitlab
+from gitlab.config import GitlabConfigMissingError
+
+from .adapter import Project
+
+DEFAULT_GITLAB_URI = 'https://gitlab.com'
+MAX_GROUPS = 999_999
+
+
+class GitlabAPI:
+    """
+    Establishes an API connection to a GitLab instance.
+    """
+
+    def __init__(self, uri=None, token=None):
+        """
+        Connects to a GitLab instance using connection details from one of the
+        local configuration files (see https://python-gitlab.readthedocs.io >
+        Configuration > Files). Connects to GitLab.com by default if no config
+        file is found. Specify an URI to override the config file lookup, the
+        token is optional (anonymous access if none is supplied).
+        """
+        if not uri:
+            try:
+                self.api = Gitlab.from_config()
+            except GitlabConfigMissingError:
+                uri = DEFAULT_GITLAB_URI
+
+        if uri:
```

Comment:
the if is redundant

Reply:
The `if` is relevant when the `GitlabConfigMissingError` is thrown.